.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+Hi, thank you for your contribution to Mantl! Before we can accept any code into
+master, we need it to meet the following criteria. If there are any you can't
+satisfy yourself, go ahead and open the issue anyway and we'll help you test.
+Feel free to delete this message once you're done. Thanks again!
+
+- [ ] Installs cleanly on a fresh build of most recent master branch
+- [ ] Upgrades cleanly from the most recent release
+- [ ] Updates documentation relevant to the changes
+- [ ] Rebases cleanly onto the latest master
```

Comment:
We need to discuss that. Are merge commits OK? We've got a really messy git history now, it's hard to tell of the intent of the PR in the main line. It'd be nice if we had all the commits in a PR collapsed into one or two logical commits, and then rebased onto master so we could avoid merge commits.